tests/lti/views/export_test.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+import urllib
+import urlparse
+
+from pyramid.view import view_config
+
+from pyramid.httpexceptions import HTTPFound
+
+from lti import util
+
+import lti.views.export
+
+def test_it_returns_HTTPFOUND(pyramid_request):
+    pyramid_request.query_string = 'args='+urllib.quote('uri=http://www.example.com&user=someone')
+    returned = lti.views.export.lti_export(pyramid_request)
+    assert isinstance(returned, HTTPFound)
+    assert returned.location == 'http://TEST_LTI_SERVER.com/export/facet.html?facet=uri&mode=documents&search=http%3A//www.example.com&user=someone'
+
+def test_it_returns_attribute_error_when_args_is_not_set(pyramid_request):
+    pyramid_request.query_string = 'something_else='+urllib.quote('uri=http://www.example.com')
```

Comment:
You can avoid repeating this line for every test by adding a `pyramid_request` fixture to this file that overrides the default `pyramid_request` fixture (defined in `conftest.py`) and adds the query string. Something like this:

```python
@pytest.fixture
def pyramid_request(pyramid_request):
    pyramid_request.query_string = 'something_else='+urllib.quote('uri=http://www.example.com')
    return pyramid_request
```
See "Fixtures override other fixtures with the same name" in [this post](https://www.seanh.cc/posts/advanced-fixtures).

Tests that use a different query string than the other tests will still have to set it themselves though.